### PR TITLE
Proofreading the Oracles Section

### PR DESCRIPTION
**Pull request by sgvictorino**
Any thoughts on these edits?
Working towards the closure of #414.

### CODE REVIEW

File: oracles.asciidoc [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
+There is a wide range of applications where the smart contract might require such extrinsic information, and it is here that *oracles* come into play. Oracles can be defined as authoritative sources of off-chain data which allow smart contracts to receive and condition their execution using this information - they can be thought of as a mechanism for bridging the gap between on and off-chain. Extending the scope of smart contracts to enforce contractual relationship based on real-world events and data broadens their scope dramatically. Examples of data that might be provided by oracles include:
```
Comment (timnugent) [inline oracles.asciidoc]: I actually think the plural use of 'range' (with 'are') is more prevalent - 'is' here sounds strange to me!

https://english.stackexchange.com/questions/19399/is-a-wide-range-of-features-singular-or-plural

File: oracles.asciidoc [hunk 2]
```diff
@@ -26,15 +26,15 @@ In this section, we will examine the primary functions of oracles, subscription
...
...ly-controlled account would transact with a decentralized application, resulting in an interaction with a function defined in the oracle smart contract. This function initiates the request to the oracle, with the associated arguments detailing the data requested in addition to supplementary information that might include callback functions and scheduling parameters. Once this transaction has been validated, the oracle request can be observed as an EVM event emitted by the oracle contract, or as a state change; the arguments can be retrieved and used to perform the actual query from the off-chain data source. The oracle may also require payment for processing the request, gas payment for the callback, and data premissions or rights obligations might have to be met. Finally, the resulting data is signed by the oracle owner, essentially attesting to the value of the data at a given time, and delivered in a transaction to the decentralized application that made the request - either directly or via the oracle contract. Depending on the scheduling parameters, the oracle may broadcast further transactions updating the data at regular intervals, for example, end of day pricing information.
```
Comment (timnugent) [inline oracles.asciidoc]: Typo - should be permissions

File: oracles.asciidoc [hunk 3]
```diff
@@ -26,15 +26,15 @@ In this section, we will examine the primary functions of oracles, subscription
...
 
-A range of alternative schemes are also possible. Data can be requested from and returned directly by an externally-controlled account, removing the need for an oracle smart contract, or the request and response can be made to and from an Internet of Things enabled hardware sensor. Oracles can therefore be human, software, or hardware-based.
+A range of alternative schemes are also possible. Data can be requested from and returned directly by an externally-controlled account, removing the need for an oracle smart contract, or the request and response can be made to and from an Internet of Things enabled hardware sensor. Oracles can, therefore, be human, software, or hardware-based.
```
Comment (timnugent) [inline oracles.asciidoc]: I think there's too many commas in this sentence now

File: oracles.asciidoc (hunk 2 above)
Comment (sgvictorino) [reply oracles.asciidoc]: Which word?

File: oracles.asciidoc [hunk 4]
```diff
@@ -56,18 +56,18 @@ If we assume that the source of data being queried by a decentralized applicatio
...
 
-There are a broad range of applications for systems like TrueBit, ranging from machine learning, to verification of any proof-of-work. An example of the latter is the Doge-Ethereum bridge, which utilizes TrueBit to verify Dogecoin’s proof-of-work, Scrypt, a memory-hard and computationally intensive function that cannot be computed within the Ethereum block gas limit. By performing this verification on TrueBit, it has been possible to securely verify Dogecoin transactions within a smart contract on Ethereum's Rinkeby testnet.
+There is a broad range of applications for systems like TrueBit, ranging from machine learning to verification of any proof-of-work. An example of the latter is the Doge-Ethereum bridge, which utilizes TrueBit to verify Dogecoin’s proof-of-work, Scrypt, a memory-hard and computationally intensive function that cannot be computed within the Ethereum block gas limit. By performing this verification on TrueBit, it has been possible to securely verify Dogecoin transactions within a smart contract on Ethereum's Rinkeby testnet.
```
Comment (timnugent) [inline oracles.asciidoc]: Same as before re: plural form of range

File: oracles.asciidoc [hunk 5]
```diff
@@ -185,7 +185,7 @@ contract OracleB1IQClient {
...
+This returns a uint256 identifier which can then be used to provide additional arguments. The addArgumentToRequestString() function is used to specify the RIC (Reuters Instrument Code), here for IBM stock, and addArgumentToRequestUint() allows the timestamp to be specified. Passing in now, an alias for block.timestamp, will, therefore, retrieve the current price for IBM. The request is then executed by the executeRequest() function. Once the request has been processed, the oracle contract will call the onSuccess callback function with the query identifier, allowing the resulting data to be retrieved, else the onFailure callback with an error code in the event of retrieval failure. The available fields that can be retrieved on success include open, high, low, close (OHLC) and bid/ask prices.
```
Comment (timnugent) [inline oracles.asciidoc]: I don't think we need these commas after therefore - standard is to do so if beginning the sentence with a therefore

https://english.stackexchange.com/questions/26498/should-there-always-be-a-comma-after-therefore-however-etc

File: oracles.asciidoc (hunk 2 above)
Comment (timnugent) [reply oracles.asciidoc]: premissions

**Comment by aantonop** [comment]
Agree on review comments. @bitsol can you edit or push a commit addressing them? With that we're ready to merge!

**Comment by sgvictorino** [comment]
How is it now @timnugent @aantonop?

**Comment by sgvictorino** [comment]
Side note: is it oracle or Oracle? It seems to be different throughout the section.

**Comment by aantonop** [comment]
I believe it's lower case, as it is not a proper noun.

**Pull request merged**